docs/content/tutorials/firewall.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: doc_page
+---
+
+What to Do When You Have a Firewall
+-----------------------------------
+When you are behind a firewall, the Maven Druid dependencies will not be accessible, as well as the IRC wikipedia channels that feed realtime data into Druid. To workaround those two challenges, you will need to:
+
+1. make the Maven Druid dependencies available offline
+2. make the Wikipedia example GeoLite DB dependency available offline
+3. use an alternative approach to load data into Druid
+
+## Making Maven Druid Dependencies Available Offline
+1. Extract Druid to a machine that has internet access; e.g. `C:\druid-<version>`
```

Comment:
agree, I think it'd be better to use something unixy for the examples. I'm not sure if anyone's actually ever got Druid working on windows.